bin/MadGraph5_aMCatNLO/cards/production/13TeV/HPlusAndH_ToWHH_ToL4B/HPlusAndH_ToWHH_ToL4B_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+import model_v4 2HDMC
+define hc = h- h+
+define w = w+ w-
+define l = l+ l-
+define v = vl vl~
+generate p p > hc h1, h1 > b b~, (hc > h1 w, h1 > b b~, w > l v)
```

Comment:
You don't want to include W -> tau decays, where the tau decays in P8 in e or mu (or jet)?

Reply:
You are right, I will include them